test/integration/eds_integration_test.cc

```diff
@@ -437,5 +438,486 @@ TEST_P(EdsIntegrationTest, StatsReadyFilter) {
   cleanupUpstreamAndDownstream();
 }
 
+class EdsOverGrpcIntegrationTest : public Grpc::EdsModeDeltaSotwIntegrationParamTest,
```

Comment:
If this class and tests do not share code with the above class/tests, I suggest refactoring it into a different file.